README.rst

```diff
@@ -101,16 +101,16 @@ Related projects
 
 .. |circleci_build| image:: https://circleci.com/gh/Deepwalker/trafaret.svg?style=shield
     :target: https://circleci.com/gh/Deepwalker/trafaret
-    :alt: Build status
+    :alt: Build status @ Circle CI
```

Comment:
Not sure about this caption.

Reply:
It's up to you. You can edit this PR right in GitHub UI and Fix whatever you need.